### PR TITLE
Blocknumberfour

### PR DESCRIPTION
**Pull request by KorinaC**
Hello Ivan, could you please take a look at this branch and tell me if there is anything I should change? Thank you!

### CODE REVIEW

File: Project materials/third client.svg
Comment (IvanCeban) [inline Project materials/third client.svg:1]: it should be png, not svg. As it is photo not vector image.
And remove spaces from all file names

File: index.html [hunk 1]
```diff
@@ -280,16 +288,129 @@
...
+            background: var(--dark-background-color, #252B42);
+            font-family: 'Montserrat';
+            display: flex;
+            padding: 0px 195px;
+            justify-content: center;
+            align-items: center;
+            align-self: stretch;
+        }
+
+        .clients-say .container {
```
Comment (IvanCeban) [inline index.html:317]: I bet you don't need to overwrite all the styles for container  for "client-say" block. If there is any difference, overwrite only it. If there is no difference in container style, just use class "container" and no need to add any other style there

**Review by IvanCeban — CHANGES_REQUESTED** [review]
Well done!
Please check images naming and formats comment. 
And container comment. The rest is fine

File: index.html (hunk 1 above)
Comment (KorinaC) [reply index.html:317]: hello Ivan, thank you for your feedback , I apllied the adjustments you suggested

**Review by IvanCeban — APPROVED** [review]
good job!

**Pull request merged**